js/clinicalTimeline.js

```diff
@@ -393,19 +394,36 @@ window.clinicalTimeline = (function(){
       .attr("x", overviewAxisWidth-3)
       .attr("y", 26)
       .attr("fill", "#ccc");
-
-    var rectangle = overviewSVG.append("rect")
-      .attr("height", 16)
-      .attr("width", rectangleOverviewWidth)
-      .attr("x", overviewX)
-      .attr("y", 27)
-      .attr("fill", "rgba(25,116,255, 0.4)")
-      .attr("class", "overview-rectangle")
-      .style("cursor", "move")
-      .call(dragRectangle);      
-
-    d3.select(divId+" svg").call(dragChart);
-    d3.selectAll(".data-control").style("visibility", "visible");
+      
+    if(advancedView){
+      var rectangle = overviewSVG.append("rect")
```

Comment:
Maybe separate this in some specific functions like initializeAdvancedView() or something of that nature